docs/admin/addons.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+
+This page lists some of the available add-ons and links to their respective installation instructions.
+
+## Networking and Network Policy
```

Comment:
Yeah, my feelings are that there is scope for "cookie-cutter" type statements about each project that give users a 1000ft view of what each does:
- X provides networking and network policy.
- Y provides networking.
- Z provides network policy.

Things like "simple, scalable, secure", "easy, fast and reliable", etc certainly rub me the wrong way (I understand that I provided some of that verbiage...), and listing the various features of each project is either going to be super verbose, or provide only a limited picture of each to the reader. 

If I were a user picking a networking option, this is not the page I'd use to make my decision, though it might be where I start looking for names to investigate.

I understand we might just be diametrically opposed here, and I don't want to be the last one holding up these docs.  Does anyone else have an opinion here?